src/sentry/workflow_engine/endpoints/validators/base/data_condition.py

```diff
@@ -40,27 +40,31 @@ def validate_condition_result(self, value: Any) -> ConditionResult:
 class BaseDataConditionValidator(
     AbstractDataConditionValidator[Any, Any],
 ):
+
+    @property
+    def condition_type(self) -> Condition:
+        if isinstance(self.initial_data, list) and self.initial_data:
+            return self.initial_data[0].get("type")
```

Comment:
`self.initial_data` is a list of conditions in some cases (e.g. the case where we're subclassing it for the `MetricIssueComparisonConditionValidator`)